app/src/main/res/values/strings.xml

```diff
@@ -85,9 +85,9 @@
     <string name="about_title_license">"License"</string>
     <string name="about_summary_license">"GNU General Public License"</string>
     <string name="offline">"You are offline"</string>
-    <string name="confirmation_authorize_now">"You need to authorize with your OSM user account to apply any of your answers for good. Authorize now?"</string>
+    <string name="confirmation_authorize_now">"You need to authorize with your OSM user account to upload your answers. Authorize now?"</string>
     <string name="not_authorized">"Not authorized"</string>
-    <string name="version_banned_message">"This version of StreetComplete has been banned from uploading any answers! Please update your version!"</string>
+    <string name="version_banned_message">"This version of StreetComplete is too old. Please update!"</string>
```

Comment:
This carries quite a different meaning though. I agree that "banning" may raise some alarm bells at the user, but it also should. The above message is only shown when a StreetComplete version is known to systematically introduce wrong data, not just because it is old.

Reply:
I think the current wording is confusing and awkward.

 The everyday user doesn’t understand (or care about) the difference between a *banned* version and an *outdated* version.

Reply:
In this case I am not sure is it necessary for user to understand why app must be updated.

For me both version are good.